src/schema/artwork/meta.js

```diff
@@ -15,8 +15,10 @@ const forSaleIndication = artwork =>
 
 const dimensions = artwork => artwork.dimensions[artwork.metric]
 
-const partnerDescription = ({ partner: { name }, forsale }) => {
-  if (!name) return undefined
+const partnerDescription = ({ partner, forsale }) => {
+  if (isEmpty(partner)) return undefined
+  const name = partner.name
```

Comment:
Can you just do `const name = partner && partner.name`?

Reply:
I'll try!

Reply:
Yup, looks like it!